mmv1/third_party/terraform/services/compute/resource_compute_region_network_endpoint_test.go.tmpl

```diff
@@ -161,7 +159,7 @@ func TestAccComputeRegionNetworkEndpoint_regionNetworkEndpointPortmapExample(t *
 
 	acctest.VcrTest(t, resource.TestCase{
 		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
-		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
```

Comment:
You'll need to remove the references to `provider = google-beta` in these configs to fix the check